LoopbackGuideApplication/build.gradle

```diff
@@ -28,6 +28,9 @@ android {
             assets.srcDirs = ['assets']
         }
 
+        defaultConfig {
+            multiDexEnabled true
+        }
```

Comment:
Please add an empty line after this one to visually delimit different configuration blocks.